api/internal/svc/rpi/service.go

```diff
@@ -26,6 +26,52 @@ func (s *Server) Ping(ctx *gin.Context) {
 	ctx.String(http.StatusOK, "pong")
 }
 
+type dailyAverage struct {
+	Value float64 `json:"value"`
+}
+
+func (s *Server) GetDailyAverageCarbonDioxide (ctx *gin.Context){
```

Comment:
Nice work! ⭐  We can start looking into adding a test for this method at some point to validate that it works correctly (however not in this PR).